src/main/java/com/github/wovnio/wovnjava/HtmlConverter.java

```diff
@@ -18,22 +18,23 @@
     HtmlConverter(Settings settings, String original) {
         this.settings = settings;
         doc = Jsoup.parse(original);
+        doc.outputSettings().indentAmount(0).prettyPrint(false);
```

Comment:
Is it needed to set `indentAmount`? Isn't it used for pretty printing only?

Reply:
I'm wrong. The option is for pretty printing only.
I removed it.
Thanks.